app/helpers/alchemy/admin/pages_helper.rb

```diff
@@ -51,6 +51,16 @@ def page_layout_missing_warning
           class: 'inline warning icon'
         )
       end
+
+      # Renders a warning icon with a hint
+      # that explains the user that their alchemy role does not
+      # allow them to make edits to this page
+      def page_roles_denial_warning
+        hint_with_tooltip(
+          Alchemy.t(:page_roles_denial),
```

Comment:
I don't think we need to introduce another translation for the same warning. The one from the controller message is sufficient.